src/components/FlawTrackers.vue

```diff
@@ -0,0 +1,454 @@
+                      class="btn py-0 dropdown-item"
+                      @click.prevent="toggleStatusFilter(status)"
+                    >
+                      <input
+                        type="checkbox"
+                        class="form-check-input me-2"
+                        :checked="statusFilter.includes(status)"
+                        @click.stop="toggleStatusFilter(status)"
+                      />
+                      <span>{{ status.toUpperCase() }}</span>
+                    </a></li>
+                  </template>
+                </ul>
+              </th>
+              <th>Resolution</th>
+              <th
+                @click="setSort('created_dt')"
+              >
+                Created date
+                <i
+                  :class="{
+                    'opacity-0': sortKey !== 'created_dt',
+                    'bi-caret-down-fill': sortOrder === descend,
+                    'bi-caret-up-fill': sortOrder !== descend,
+                  }"
+                  class="bi"
+                />
+              </th>
+              <th
+                @click="setSort('updated_dt')"
+              >
+                Updated date
+                <i
+                  :class="{
+                    'opacity-0': sortKey !== 'updated_dt',
+                    'bi-caret-down-fill': sortOrder === descend,
+                    'bi-caret-up-fill': sortOrder !== descend,
+                  }"
+                  class="bi"
+                />
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              v-for="(tracker, trackerIndex) in paginatedTrackers"
+              :key="trackerIndex"
+            >
+              <td>
+                <RouterLink :to="{ path: `/tracker/${tracker.uuid}` }">
+                  {{ `${tracker.external_system_id} ` }}<i class="bi-box-arrow-up-right" />
+                </RouterLink>
+              </td>
+              <td>{{ tracker.ps_update_stream }}</td>
+              <td>{{ tracker.ps_module }}</td>
+              <td>{{ tracker.status?.toUpperCase() || 'EMPTY' }}</td>
+              <td>{{ tracker.resolution?.toUpperCase() }}</td>
+              <td>{{ formatDate(tracker.created_dt ?? '', true) }}</td>
+              <td>{{ formatDate(tracker.updated_dt ?? '', true) }}</td>
+            </tr>
+          </tbody>
+        </table>
+        <div v-if="sortedTrackers.length === 0">
+          No trackers to display
+        </div>
+      </div>
+    </div>
+    <TrackerManager
+      v-if="showTrackerManager"
+      mode="embedded"
+      :flawId="flawId"
+      :affects="affectsNotBeingDeleted"
+      @affects-trackers:refresh="emit('affects:refresh')"
+    />
+  </div>
+</template>
+
+<style scoped lang=scss>
+@import '@/scss/bootstrap-overrides';
+
+.affects-trackers {
+  .pagination-controls {
+    display: flex;
+
+    button {
+      height: 2rem;
+      padding-block: 0;
+    }
+
+    .page-btn {
```

Comment:
```suggestion
    .osim-page-button {
```
to distinguish from bootstrap more